### DISCUSSION TITLE
Tree: How to disable node highlighting on right-click?

### DISCUSSION THREAD

**Discussion by ghost** [primefaces/community#2784] [PrimeReact] [answered]
### Describe the bug

How to disable the highlight that appears in the node when right clicking? 

See the following fork: 

https://stackblitz.com/edit/ltbytg-vcxpee?file=src%2FApp.jsx

I can't disable such highlight using context menu events:

![image](https://github.com/user-attachments/assets/4c028480-fd26-4e1e-be98-e70c541449f0)

Thank you.

### Reproducer

https://stackblitz.com/edit/ltbytg-vcxpee?file=src%2FApp.jsx

### System Information

```Shell
npmPackages:
    primereact: ^10.8.2 => 10.8.2
    react: ^18.3.1 => 18.3.1
```

### Steps to reproduce the behavior

1. Go to the forked project.
2. Right-click in any node, the highlight is always there.

### Expected behavior

There should be a possibility to disable such highlight.

**Comment by melloware** [comment]
@VentuzVictorPerez moved this to community discussion as its not a bug for the tracker.

**Comment by qwadrox** [comment, marked as answer]
@VentuzVictorPerez Hi! There is no such option because you can control the visibility with simple CSS:

```css
.p-tree .p-tree-container .p-treenode:focus > .p-treenode-content {
  box-shadow: none;
}

.p-tree .p-treenode-content.p-highlight {
    background: none;
}
```
https://stackblitz.com/edit/ltbytg-wzbynw?file=src%2Findex.css

We can always use CSS to control subtle changes in the component appearances. 
If you require, there is also a [theme designer](https://primereact.org/theming/#customtheme), which gives you the flexibility to modify all aspects of the component style. I recommend you take a look. You can find tutorial on YouTube how to get start with it.
